site/src/components/pages/docs/search/index.tsx

```diff
@@ -64,36 +65,39 @@ export const Search: React.VoidFunctionComponent<SearchProps> = ({
     "normal" | "noresults" | "failed"
   >("normal");
 
-  const searchOnlineDebounce = useMemo(
-    () =>
-      debounce((newSearchText: string) => {
-        setSearchLoading(true);
-        index
-          .search<AlgoliaResult>(newSearchText)
-          .then(({ hits }) => {
-            setSearchState(hits.length === 0 ? "noresults" : "normal");
-            setSearchResults(hits.slice(0, MAX_SEARCH_RESULTS));
-            setActiveResult(MAX_SEARCH_RESULTS);
-            setCurrentSearchedText(newSearchText);
-            setSearchLoading(false);
-          })
-          .catch((err) => {
-            // @todo use logger later
-            // eslint-disable-next-line no-console
-            console.error(err);
-            setSearchState("failed");
-            setSearchLoading(false);
-          });
-      }, 500),
-    [],
+  const searchOnlineDebounce = useRef(
```

Comment:
Just for my understanding, what's the thinking behind a `useMemo` in a `useRef`?

Reply:
Addressed in https://github.com/blockprotocol/blockprotocol/pull/396/commits/fad381236a1f8b814b57988d5d37006adaac7f53